client/src/index.js

```diff
@@ -4,7 +4,7 @@ import { Router as BrowserRouter } from 'react-router-dom';
 import { createBrowserHistory } from 'history';
 
 import './index.scss';
-import App from './App';
+import App from './App.tsx';
```

Comment:
I'm sure there's gotta be a way to configure the app so that we don't need to type file extensions every time. But I don't know what that is.